### ISSUE TITLE
Incoming call sound doesn't play

### ISSUE THREAD

**Issue by Overlord23** [notsecure/uTox#1045]
In Utox v0.3.0 the incoming call ring doesn't play when a call is been received.
There are two sound adapters on my computer, the Realtek onboard audio subsystem and a Creative Audigy SE PCI.
Even switching from one to another, the sound isn't been reproduced.

**Comment by irungentoo** [comment]
does disabling audio filtering on the sender fix it?

**Comment by Overlord23** [comment]
I just tried that, the problem persist.
I tried also switching sound adapter with audio filtering disabled either on sender and receiver but it's the same.

**Comment by irungentoo** [comment]
if you restart your client does it fix it?

**Comment by GrayHatter** [comment]
did you enable it in the setting?

and are you sure your volume is up?

**Comment by Overlord23** [comment]
In the settings menu the related option is enabled and all volume triggers are properly set, other applications can play any kind of sound on both audio subsystems.

Then I set for both input and output device the Realtek subsystem; in this way the ringtone plays as normally, however if I switch output device to "Creative Audigy", this issue appears again; even after restarting uTox the ringtone doesn't play.
I'm working on Windows 7 Pro x64.

After that I disabled the Realtek output device from Windows audio settings, so that only the Creative audio subsystem is enabled.
I restarted uTox and now the ringtone has been played.

The problem appears only when multiple audio subsystems are available.
